Demo.ipynb

```diff
@@ -201,7 +199,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.5"
+   "version": "3.7.9"
```

Comment:
Is changing the version here just specifying the version of python3? Is there a big difference between 3.8.5 and 3.7.9?

Reply:
I don't believe there are major differences, though I know there are some. I don't think the changes impacted this code though. I should probably just update my environment. This change must have occurred because my Anaconda environment (which is how I was running Jupyter notebook) is version 3.7.9, whereas your environment is likely version 3.8.5.

Reply:
Yeah, it's neat to see that! Python version 3.8 is better, I just haven't updated Anaconda recently. If we wanted to get really fancy, we could try to implement some form of version control (i.e., dictate the python and package versions that people must use if they want to be sure that this will work). That is probably overkill for what we are doing, but you will almost certainly see that on larger open-source projects (including [Breakthrough Energy's packages](https://breakthrough-energy.github.io/docs/)).